.gitignore

```diff
@@ -0,0 +1,42 @@
+#binaries
+mkassets
+proxima5
+
+#generated assets
```

Comment:
While these assets are generated, the generated results are intended to be under git.  Not everyone should need to regenerate them every time.  Please don't include them here.

Reply:
It makes sense! I'll add them as well